external/fv3fit/fv3fit/reservoir/domain2.py

```diff
@@ -54,10 +54,14 @@ def __init__(
         if overlap < 0:
             raise ValueError("Overlap must be non-negative")
 
+        self._rank_dims = ["x", "y"]
```

Comment:
These were hardcoded since everything is set up to transpose to this form.  Needed since the partitioner expects `[y, x]` shape.  Maybe in the long-run it would be easier to switch to `[y, x]` since that's the default in the wrapper runtime.

Reply:
I ran into this issue with the wrapper dim order being different, I think currently this is handled by  transposing to the wrapper's order  in the adapter.